### PR TITLE
PLAT-430 Added ability to handle values from HTTP headers.

### PR DESCRIPTION
**Pull request by pies**

### CODE REVIEW

File: src/RuntimeSettings.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    /**
+     * @param string $name
+     * @param string $default
+     * @return string
+     */
+    public function getValue($name, $default = null)
+    {
+        $header = self::getHeader($name);
```
Comment (piotrjura) [inline src/RuntimeSettings.php]: Why self::, is there anything static here?

File: src/ConfigurationProvider.php [hunk 2]
```diff
@@ -66,67 +60,11 @@ public function register(Application $app)
...
-     */
-    public static function getTenant()
-    {
-        return self::getValue(self::TENANT_KEY, self::TENANT_DEFAULT);
-    }
-
-    /**
      * @return bool
      */
     public static function isDev()
```
Comment (piotrjura) [inline src/ConfigurationProvider.php:66]: BTW, where is this static method of provider used?

**Comment by piotrjura** [comment]
:+1:

File: src/ConfigurationProvider.php (hunk 2 above)
Comment (pies) [inline src/ConfigurationProvider.php:66]: This is for backwards compatibility, I'll add a `@deprecated` comment.

**Pull request merged**